src/file.ts

```diff
@@ -3517,6 +3519,8 @@ class File extends ServiceObject<File> {
    * Resumable uploads are automatically enabled and must be shut off explicitly
    * by setting `options.resumable` to `false`.
    *
+   * Multipart uploads with retryable error codes will be retried 10 times with exponential backoff.
+   *
    * <p class="notice">
    *   There is some overhead when using a resumable upload that can cause
```

Comment:
Out of curiosity, is this true? The node library doesn't automatically choose multipart for small files?

Reply:
It does not have a short circuit on that. It depends on a config option named `resumable` to be set to false for a multipart. By default it's always true (resumable)